frontend/public/components/markdown-view.tsx

```diff
@@ -139,13 +143,18 @@ export class SyncMarkdownView extends React.Component<
         content || emptyMsg || 'Not available',
       )}</div></body>`;
     return (
-      <iframe
-        sandbox="allow-popups allow-same-origin"
-        srcDoc={contents}
-        style={{ border: '0px', display: 'block', width: '100%', height: '0' }}
-        ref={(r) => (this.frame = r)}
-        onLoad={() => this.updateDimensions()}
-      />
+      <>
+        <iframe
+          sandbox="allow-popups allow-same-origin"
+          srcDoc={contents}
+          style={{ border: '0px', display: 'block', width: '100%', height: '0' }}
+          ref={(r) => (this.frame = r)}
+          onLoad={() => this.updateDimensions()}
+        />
+        {this.frame?.contentDocument && (
+          <MarkdownHighlightExtension docContext={this.frame.contentDocument} />
```

Comment:
setup a `key` incase the `content` changes so that we unmount and re-mount the extensions

Reply:
done